AGENTS.md

```
@@ -0,0 +1,129 @@
+npm run ios               # expo start --ios
+npm run android           # expo start --android
+npm run web               # expo start --web
+```
+
+## Architecture
+
+### Monorepo layout
+
+Published npm package: **`@ant-design/react-native`**. Source lives under `components/`; build artifacts go to `lib/` (CJS) and `es/` (ESM). **Never edit `lib/` or `es/` directly** — change `components/` and run `npm run compile`.
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)